maven/jdk8/Dockerfile

```diff
@@ -1,22 +1,37 @@
 # Use Debian base
 FROM jenkins/inbound-agent:jdk8 as jnlp
 
+FROM eclipse-temurin:11.0.16_8-jdk-focal AS jre11-build
+# Generate smaller java runtime without unneeded files
+# for now we include the full module path to maintain compatibility
+# while still saving space (approx 200mb from the full distribution)
+RUN jlink \
+    --add-modules ALL-MODULE-PATH \
+    --strip-debug \
+    --no-man-pages \
+    --no-header-files \
+    --compress=2 \
+    --output /javaruntime
+
 FROM maven:3.8.6-eclipse-temurin-8
 
 ## Ignore apt package pinning as we always want the latest package here
 # hadolint ignore=DL3008
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        bash \
-        git \
-        libfontconfig1 \
-        libfreetype6 && \
+    bash \
+    git \
+    libfontconfig1 \
+    libfreetype6 && \
```

Comment:
I think the original indentation was more legible: these are all args to `apt-get install` rather than executables listed for `RUN`.

Reply:
Looks like my docker plugin in vscode has a new feature...

Reply:
could probably be fixed by combining update and the install line into one, then each package would be aligned properly

Reply:
Lemme remove the indentation, to ease review